bin/common.sh

```diff
@@ -128,9 +133,9 @@ function for_each_cluster() {
   done
 }
 
-hf_templates_init() {
+function hf_templates_init() {
   local out_dir="$1"
   shift
-  [ -z "$*" ] && hf -f helmfile.tpl/helmfile-init.yaml template --skip-deps --output-dir="$out_dir" >/dev/null 2>&1
-  hf "$@" template --skip-deps --output-dir="$out_dir" >/dev/null 2>&1
-}
+  [ -n "$all" ] && hf -f helmfile.tpl/helmfile-init.yaml template --skip-deps --output-dir="$out_dir" >/dev/null 2>&1
```

Comment:
Hmm.. actually not too sure whether we should just leave `[ -z "$*" ]` here. Any input?

Reply:
I believe Alin added that to exclude that line when labels are given, but it never really worked I think. Anyways, that should be solved by this PR.

Reply:
If this is the use-case, I will retrofit it to exclude that line when labels are given.